events.html

```diff
@@ -2,13 +2,16 @@
 layout: default
 title: Events
 section_id: events
+cover-pic: /images/group/denero-lilt-cropped.jpg
+
+custom_css: shared.css
 ---
 
-<div class="full">
+<div class="full" style="background-image: url({{page.cover-pic}})">
   <div class="row">
     <div class="large-12 columns">
```

Comment:
Can we possibly change this to use the same div as the one on the indrel page? (<div class="title-section cover ...">)